### PR TITLE
Adding Fedora installation steps

### PR DESCRIPTION
**Pull request by ruzickap**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -46,7 +46,10 @@ xXl2Nm/u3cPP/eQVrZz5H8eACwIv+LL1EV+9uLanWUa+IO5hHr3KElvKNKD6vN0=
 
 ### GNU Compiler
 ```
+    # Ubuntu / Debian
     sudo apt-get install libmicrohttpd-dev libssl-dev cmake build-essential libhwloc-dev
+    # Fedora
```
Comment (psychocrypt) [inline README.md]: If you are adding fedora here you need to change line 45 and remove the tezt in the breakets

**Comment by psychocrypt** [comment]
Please rebase your pull request against dev. You could force push to this branch after the rebase and the pull request is updated automaticly.

File: README.md (hunk 1 above)
Comment (ruzickap) [inline README.md]: I missed that - sorry.
It should be fixed now.

**Comment by fireice-uk** [comment]
@psychocrypt I think this needs libhwloc-dev equivalent on Fedora. Otherwise, is this ok to merge? 

Since we are doing distro-galore I will add Arch build instructions too =).

**Comment by ruzickap** [comment]
The steps should be fine... as far as I can test...

**Comment by fireice-uk** [comment]
@ruzickap You are correct - different order confused me.

**Comment by psychocrypt** [comment]
@fireice-uk  as I can see @ruzickap added the devel for CentOS and Fedora.
I can't verify the install steps but all in all it looks valid.

**Pull request merged**